S/SDL2_ttf/bundled/patches/configure_in-v2.0.15.patch

```diff
@@ -0,0 +1,53 @@
+--- a/configure.in
++++ b/configure.in
+@@ -64,6 +64,7 @@
+ 	;;
+ esac
+ 
++PKG_PROG_PKG_CONFIG
+ 
+ case "$host" in
+     *-*-beos*)
+@@ -87,23 +88,30 @@
+ AM_CONDITIONAL(USE_VERSION_RC, test x$use_version_rc = xtrue)
+ 
+ dnl Check for the FreeType 2 library
+-PKG_CHECK_MODULES([FT2], [freetype2 >= 7.0.1], [], [dnl
+-    AC_CHECK_FT2(,,[AC_MSG_ERROR([dnl
+-*** Unable to find FreeType2 library (http://www.freetype.org/)])]
+-    )
+-])
+-CFLAGS="$CFLAGS $FT2_CFLAGS"
+-LIBS="$LIBS $FT2_LIBS"
++PKG_CHECK_MODULES(
++	FREETYPE2,
++	freetype2,
++	[
++		ft_found=yes
++		CFLAGS="$CFLAGS $FREETYPE2_CFLAGS"
++		LIBS="$LIBS $FREETYPE2_LIBS"
++	],
++	ft_found=no
++)
+ 
+ dnl Check for SDL
+ SDL_VERSION=2.0.8
+ AC_SUBST(SDL_VERSION)
+-AM_PATH_SDL2($SDL_VERSION,
+-            :,
+-	    AC_MSG_ERROR([*** SDL version $SDL_VERSION not found!])
++PKG_CHECK_MODULES(
++	SDL2,
++	sdl2,
++	[
++		sdl2_found=yes
++		CFLAGS="$CFLAGS $SDL22_CFLAGS"
++		LIBS="$LIBS $SDL2_LIBS"
++	],
++	sdl2_found=no
+ )
+-CFLAGS="$CFLAGS $SDL_CFLAGS"
+-LIBS="$LIBS $SDL_LIBS"
```

Comment:
@giordano Is there any reason why you changed these variable names?: `SDL_LIBS` to `SDL2_LIBS` and `SDL_CFLAGS` to `SDL22_CFLAGS`?

It looks like probably a find-and-replace mistake to me?
I'm wondering if somehow because you're not specifying the libs here, maybe it's defaulting to "all the libs it can find" or something dumb, and that's why it's picking up `SDL2main` from SDL2_jll?

Reply:
About the why I changed the name of the variables: I think that the way the default `configure` script was looking for SDL configuration was broken with our setup, I just switched to using the pkgconfig file, and since the module is called "SDL2" (see the first argument of `PKG_CHECK_MODULES`) the variables names will start with "SDL2".